src/blog/2022/07/new-projecttype.md

```diff
@@ -0,0 +1,23 @@
+---
+title: New Project Type
+subtitle: A bigger project with more resources
+description: A bigger project with more resources
+date: 2022-07-22 12:00:00.0
+authors: ["sam-machin"]
+---
+We've added a second size of project to our offering on FlowForge Cloud
+<!--more-->
+
+Our [0.7 release](https://flowforge.com/blog/2022/07/flowforge-07-released/) introduced the concept of Project Types, this allows us to offer projects at different price points based on the stack they are using.
+Today we've put this feature to work by introducing the medium sized project alongside the existing small size, you will be offered the choice of types when you create a new project.
+
+![](../images/project-type.png)
+
+Medium projects have 3 times the resources of the small allowing for more complex flows and larger message objects, this will be useful to business users looking to process complex sets of data.
+Our Medium project is priced at $50 a month, we'll be adding additional new features to this project type in the coming months to further enhance the value of the higher tier.
```

Comment:
```suggestion
Medium projects have 3 times the resources of the existing Small type, allowing for more complex flows and larger message objects. This will be useful to business users looking to process complex sets of data.

Our Medium project is priced at $50 a month and we'll be adding new features to this project type in the coming months to further enhance the value of this new tier.
```